pilot/pkg/model/telemetry_test.go

```diff
@@ -243,6 +243,78 @@ func TestAccessLogging(t *testing.T) {
 	}
 }
 
+func TestAccessLoggingWithFilter(t *testing.T) {
+	sidecar := &Proxy{ConfigNamespace: "default", Metadata: &NodeMetadata{Labels: map[string]string{"app": "test"}}}
+	filter1 := &tpb.Telemetry{
+		AccessLogging: []*tpb.AccessLogging{
+			{
+				Providers: []*tpb.ProviderRef{
+					{
+						Name: "custom-provider",
+					},
+				},
+				Filter: &tpb.AccessLogging_Filter{
+					Expression: "response.code >= 400",
```

Comment:
Also in line with the other comment - expressions like this may be an anti-pattern (although perfectly fine for test case here). This would miss `code == 0`, which indicates a failure as well

Reply:
Here is a [cel demo](https://github.com/zirain/playground/blob/master/go/cel/main.go), `code == 0` is valid.

Reply:
updated with a simple webhook implement, PTAK